data/sql_setup/prepare_schedule_e.sql

```diff
@@ -217,7 +217,7 @@ end
 $$ language plpgsql;
 
 drop trigger if exists ofec_sched_e_queue_trigger on fec_vsum_sched_e;
-create trigger ofec_sched_e_queue_trigger before insert or update or delete
+create trigger ofec_sched_e_queue_trigger instead of insert or update or delete
```

Comment:
We want all record processing to go directly against the queue tables instead of the source table?  My understanding of `instead of` means that any record change that is intended to happen against the `fec_vsum_sched_e` will happen only in the queue tables now with this change.  I just wanted to make sure that this is the desired behavior (and I think we talked about this before, but might be blanking on a detail from that conversation!).